AnkiDroid/src/main/java/com/ichi2/anki/preferences/HeaderFragment.kt

```diff
@@ -48,6 +57,31 @@ class HeaderFragment : PreferenceFragmentCompat() {
         )
     }
 
+    private fun highlightHeaderPreference(headerPreference: HeaderPreference) {
+        if (!(activity as Preferences).hasLateralNavigation()) {
+            return
+        }
+        selectedHeaderPreference?.setHighlighted(false)
+        selectedHeaderPreference = headerPreference
+        selectedHeaderPreference?.setHighlighted(true)
+        selectedHeaderPreferenceKey = headerPreference.key
```

Comment:
```suggestion
        selectedHeaderPreference?.setHighlighted(false)
        // highlight the newly selected header
        selectedHeaderPreference = headerPreference.apply {
            setHighlighted(true)
            selectedHeaderPreferenceKey = this.key
        }
```

I feel this is a little more readable